CHANGELOG.md

```diff
@@ -3,6 +3,24 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# [1.6.0](https://github.com/adobe/spectrum-web-components/compare/v1.5.0...v1.6.0) (2025-05-01) — ([f2b6a32](https://github.com/adobe/spectrum-web-components/commit/f2b6a322c379752f4f7dfcd02b3007c4130dc1e0))
+
+### Minor Fixes
+
+-   **icons-workflow**: added missing S2 icons
+
+### Bug Fixes
```

Comment:
If we decide to go ahead with changesets way I feel keeping the grouping as `Major, Minor and Patch` make sense. But if we need to change to a more custom group like bug fixes this will require additional customizations. 
Please use this script in this PR to automatically generate this global CHANGELOG.
https://github.com/adobe/spectrum-web-components/pull/5328

Reply:
We tried that but it has to be ran before release which we didn’t know so the output was null. I manually drafted this for this release. We can change bug to patch I was just following the previous release patterns